src/qt/transactionview.cpp

```diff
@@ -592,6 +591,21 @@ void TransactionView::focusTransaction(const QModelIndex &idx)
     transactionView->setFocus();
 }
 
+void TransactionView::focusTransaction(const uint256& txid)
+{
+    if(!transactionProxyModel)
+        return;
+
+    const QModelIndexList results = this->model->getTransactionTableModel()->match(
+        this->model->getTransactionTableModel()->index(0,0),
+        TransactionTableModel::TxHashRole,
+        QString::fromStdString(txid.ToString()));
+
+    if (results.isEmpty()) return;
+
+    focusTransaction(results[0]);
```

Comment:
We support selecting multiple transactions, so this ought to select all of them...

Reply:
I consider more than 1 result an error, given that we're passing a single tx id. I could replace the `isEmpty()` with a check that length is 1.

Reply:
It's definitely not an error. The transaction list shows logical transactions, not blockchain transactions. There is 1 txid per blockchain transaction, but that could be many logical transactions.

Reply:
Ah, I see what you mean. E.g. sending to two destinations in one transaction results in two rows in the transactions tab.

Reply:
@luke-jr :+1: